cmd/brew-test-bot.rb

```diff
@@ -984,7 +984,7 @@ def test_ci_upload(tap)
     # These variables are for Jenkins and Circle CI respectively.
     pr = ENV["UPSTREAM_PULL_REQUEST"] || ENV["CIRCLE_PR_NUMBER"]
     if pr
-      pull_pr = "#{tap.remote}/pull/#{pr}"
+      pull_pr = "#{tap.remote.sub(/\.git$/, "")}/pull/#{pr}"
```

Comment:
Correct. Though I am curious why Homebrew supports atypical git remotes and `homebrew-test-bot` does not, I can live without this functionality.